addons/point_of_sale/static/tests/generic_helpers/offline_util.js

```diff
@@ -0,0 +1,33 @@
+import { run } from "@point_of_sale/../tests/generic_helpers/utils";
+import { ConnectionLostError } from "@web/core/network/rpc";
+
+const originalFetch = window.fetch;
+const originalConsoleError = console.error;
+const originalSend = XMLHttpRequest.prototype.send;
```

Comment:
can you also do the same for `pos.onNotified` ? You can access pos here using `/* global posmodel */`